our_site/planner/views.py

```diff
@@ -1,33 +1,65 @@
 from django.http import HttpResponse, HttpResponseRedirect
 from django.template import loader
+from .models import Event
 import django.forms as forms
+from django.shortcuts import render
 
 
-class DashboardFilterAllEvents(forms.Form):
-    pass
 
+def dashboard(request): 
```

Comment:
Trying to get the profile before the user is known to be anonymous or not causes the website to crash when an anonymous user attempts to view a dashboard instead of redirecting them to the login page.